google/cloud/apigee/registry/applications/v1alpha1/diff_analytics.proto

```diff
@@ -43,3 +43,15 @@ message ClassifiedChanges {
   Diff unknownChanges = 3;
 }
 
+message Stats {
+  int64 totalChanges = 1;
+  int64 totalBreakingChanges = 2;
+  int64 totalNonBreakingChanges = 3;
+  int64 numberOfDiffs = 4;
```

Comment:
nit: "numberOfDiffs" is more verbose than I'm used to, in particular, I haven't seen "of" included in any APIs. Generally something like "num_diffs" would suffice, but if there's precedent for this in our external code, please feel free to ignore.

Reply:
Done

Reply:
[AIP-141](https://google.aip.dev/141) actually addresses this. Maybe this should be `change_count`, `breaking_change_count`, `nonbreaking_change_count`, and `diff_count`?